### PR TITLE
Move file upload api into sub module

### PR DESCRIPTION
**Pull request by PandekoAndrew**

### CODE REVIEW

File: socnet-file-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+version '0.0.1-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+ext {
+    apacheCommonsLangVersion = '2.6'
```
Comment (DenSev) [inline socnet-file-service/build.gradle]: ext variables can be left in the main build.gradle file.

File: socnet-file-service/build.gradle [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+
+group 'com.example'
+version '0.0.1-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
```
Comment (DenSev) [inline socnet-file-service/build.gradle]: Can be left in the main build.gradle file.

File: socnet-file-service/build.gradle (hunk 1 above)
Comment (PandekoAndrew) [reply socnet-file-service/build.gradle]: done

File: socnet-file-service/build.gradle (hunk 2 above)
Comment (PandekoAndrew) [reply socnet-file-service/build.gradle]: done

**Pull request merged**